### PR TITLE
BOAC-1499, Rainy days and Mondays (and blue menu items) always get me down

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/BOAC-1499

### CODE REVIEW

**Comment by johncrossman** [comment]
Do not merge. A similar fix is needed on pagination-widget.

**Comment by johncrossman** [comment]
Okay, additional fix is in. Travis is chewing on it.

File: boac/static/app/main.css [hunk 1]
```diff
@@ -90,6 +90,15 @@
...
 
+.dropdown-menu li a,
+.pagination li a {
+  color: #333 !important;
+}
+
+.pagination .active a {
+  color: #fff !important;
+}
+
```
Comment (johncrossman) [inline boac/static/app/main.css]: The cause: [`#content a` in PR 1042](https://github.com/ets-berkeley-edu/boac/pull/1042/files#diff-62ebc88537e133e0d5249892d34c03d3R380). These overrides will be tossed out when we leave Angular bootstrap.

**Comment by paulkerschen** [comment]
How does it feel, to override my menu?

**Pull request merged**